recipe/meta.yaml

```diff
@@ -14,10 +14,12 @@ source:
     - icu4c-4_9_1-mingw-w64-mkdir-compatibility.patch  # [win]
     # Omit lib prefix on libraries.
     - icu-config.patch  # [win]
+    # configure uses /W4 when it shouldn't which breaks things (ie link steps)
+    # Given we aren't interested in warnings, remove it
+    - configure_msvc.patch  # [win]
 
 build:
-  number: 0
-  skip: True  # [win and py36]
```

Comment:
I guess that the reason we have this is to avoid building `vc14` twice since the Python 3.5 version will be OK for Python 3.6 too (and since there is no `run` dependency on python there is no Python build string, only the `vc`. ).